gapis/api/sync/data.go

```diff
@@ -24,85 +24,43 @@ import (
 // synchronization dependency.
 type SynchronizationIndices []api.CmdID
 
-// The index of a subcommand within a command
-type SubcommandIndex []uint64
-
 // ExecutionRanges contains the information about a blocked command.
 // LastIndex is the final subcommand that exists within this command.
 // Ranges defines which future command will unblock the command in question, and
 // which subcommandis the last that will be run at that point.
 type ExecutionRanges struct {
-	LastIndex SubcommandIndex
-	Ranges    map[api.CmdID]SubcommandIndex
+	LastIndex api.SubCmdIdx
+	Ranges    map[api.CmdID]api.SubCmdIdx
+}
+
+// SubcommandReference contains a subcommand index as well as an atom.ID that
+// references the command that generated this subcommand.
+type SubcommandReference struct {
+	Index          api.SubCmdIdx
+	GeneratingAtom api.CmdID
```

Comment:
Please rename to `GeneratingCmd`. I'm on a mission to expunge atom from the codebase.